notebooks/WFC3/point_spread_function/hst_point_spread_function.ipynb

```diff
@@ -0,0 +1,1551 @@
+{
```

Comment:
<p>Your notebook is well-structured and clear. For directory creation logic, I recommend using <code>if not os.path.exists(folder):</code> to check dir before creating it. This makes the intent more explicit.</p><p> </p>

<pre class="ql-syntax" spellcheck="false">for folder in ['data', 'psfs', 'plot', 'driz', 'tweak']:
&nbsp; &nbsp; if not os.path.exists(folder):
&nbsp; &nbsp; &nbsp; &nbsp; os.mkdir(folder)
&nbsp; &nbsp; else:
&nbsp; &nbsp; &nbsp; &nbsp; print(f'{folder} directory exists.')
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/hst_notebooks/pull/270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
Agreed and implemented.